modules/passkey/test/4337/SafeWebAuthnSigner.spec.ts

```diff
@@ -1,300 +0,0 @@
-import { expect } from 'chai'
```

Comment:
Moved to the `Safe4337Module.spec.ts` spec, alongside the equivalent `SafeWebAuthnSharedSigner` tests.